versions/3.0.md

```diff
@@ -1050,6 +1051,36 @@ Response with no return value:
 description: object created
 ```
 
+#### <a name="callbackObject"></a>Callback Object
+
+A container for possible out-of band callbacks from an operation. A callback may be returned from an operation, calling back to the path specified in the operation object.
```

Comment:
»A callback may be returned from an operation« sounds wrong. Maybe better »A callback may be caused/triggered by an operation«?